published/011-2018-sep-26.md

```diff
@@ -0,0 +1,62 @@
+## Welcome back to the IPFS Weekly. 👋
+
+The [InterPlanetary File System (IPFS)](https://ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities. IPFS enables the creation of completely distributed applications. It aims to make the web faster, safer, and more open. Since that’s a pretty large scope, we track development across the ecosystem in this weekly dispatch.
+
+Looking to get involved? Click on some of the links below, see what we’re up to on [GitHub](https://github.com/ipfs), or join us on [IRC](https://webchat.freenode.net/?channels=ipfs_) (#ipfs on the Freenode network).
+
+Want this update in your inbox? [Subscribe to our weekly newsletter!](https://tinyletter.com/ipfsweekly)
```

Comment:
This should be `https://tinyletter.com/ipfsnewsletter`. This URL is 404ing which is causing the build failure.